pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
for first release I am ok with 1 jar, but there will be a need to split this into multiple jars:  for example https://issues.apache.org/jira/browse/CASSANDRA-15602